python/yugabyte_db_thirdparty/build_definition_helpers.py

```diff
@@ -24,6 +24,7 @@
     remove_path,
     copy_file_and_log
 )
+from yugabyte_db_thirdparty.arch import is_macos_arm64_build
```

Comment:
Do we need this one?

Reply:
Yes, we include this file from each build definition file and we use this function in a couple of them.